samples/word/99-preview-apis/manage-comments.yaml

```diff
@@ -0,0 +1,235 @@
+id: word-manage-comments
+name: Manage comments
+description: Basic operations using comments.
+host: WORD
+api_set:
+    WordApi: '1.4'
+script:
+    content: |
+        $("#insert").click(() => tryCatch(insertComment));
+        $("#reply").click(() => tryCatch(replyToFirstCommentInSelection));
+        $("#get-comments").click(() => tryCatch(getComments));
+        $("#get-comments-in-selection").click(() => tryCatch(getCommentsInSelection));
+        $("#edit").click(() => tryCatch(editFirstCommentInSelection));
+        $("#resolve").click(() => tryCatch(resolveFirstCommentInSelection));
+        $("#delete").click(() => tryCatch(deleteFirstCommentInSelection));
+        $("#range").click(() => tryCatch(getFirstCommentRange));
+        $("#setup").click(() => tryCatch(setup));
+
+        async function insertComment() {
```

Comment:
As per our recent code comment discussion, let's add comments before each method. Here's an Excel example to help with consistency: https://github.com/OfficeDev/office-js-snippets/blob/main/samples/excel/42-range/precedents.yaml

Reply:
Updated